frontend/src/utils/personUtils.ts

```diff
@@ -4,7 +4,9 @@ export function formatFullName(person?: Partial<IEditablePerson>): string {
   if (!person) {
     return '';
   }
-  const nameParts = [person.firstName, person.middleNames, person.surname];
-  const fullName = nameParts.filter(n => n != null && n.trim() !== '').join(' ');
-  return fullName;
+  return formatNames([person.firstName, person.middleNames, person.surname]);
+}
+
+export function formatNames(nameParts: Array<string | undefined | null>): string {
+  return nameParts.filter(n => n !== null && n !== undefined && n.trim() !== '').join(' ');
```

Comment:
Not sure I like this change. I rather have an interface that explicitly calls what name parts are we using (first, middle, last) than to rely on the position of the string in the incoming array. Thoughts?

Reply:
NVM, I see you refactor the above method to call this one, should be fine

Reply:
This one only concatenates the string, it provides the flexibility to change the order if needed be